Makefile

```diff
@@ -134,7 +134,7 @@ compile_translations: # compile translation files, outputting .po files for each
 fake_translations: ## generate and compile dummy translation files
 
 pull_translations: ## pull translations from Transifex
-	tx pull -t -af --mode reviewed
+	tx pull -t -a -f --mode reviewed
```

Comment:
Is this a change you meant to include? Or is it from another commit?

Reply:
This is definitely from another commit. Probably due to rebase stuff (again) but I swear I did it correctly this time. 

I ran my commands in this order when opening up VSCode (starting in my branch):
```
git stash
git checkout main
git pull
git checkout [my branch]
git rebase main
git stash pop
```

Here's my reflog for that stuff for reference (I believe I rebased on two different occasions):
```
acfb8ea HEAD@{15}: rebase (finish): returning to refs/heads/ilee2u/submit-on-timeout
acfb8ea HEAD@{16}: rebase (start): checkout main
beb4409 HEAD@{17}: checkout: moving from main to ilee2u/submit-on-timeout
acfb8ea HEAD@{18}: pull: Fast-forward
beb4409 HEAD@{19}: checkout: moving from ilee2u/submit-on-timeout to main
beb4409 HEAD@{20}: rebase (finish): returning to refs/heads/ilee2u/submit-on-timeout
beb4409 HEAD@{21}: rebase (start): checkout main
9f7be01 HEAD@{22}: checkout: moving from main to ilee2u/submit-on-timeout
beb4409 HEAD@{23}: pull: Fast-forward
9f7be01 HEAD@{24}: checkout: moving from ilee2u/feat-attempt-polling-endpoint to main
```